src/Entities/Player.cpp

```diff
@@ -213,6 +213,7 @@ cPlayer::~cPlayer(void)
 	SaveToDisk();
 
 	m_ClientHandle = nullptr;
+	SetIsTicking(false);
```

Comment:
This doesn't look right, you're essentially just disabling the assert here.

I think it should probably be called as soon as the the disconnect packet is received. Or, possibly when the player is actually removed from the world.

Reply:
Okay, one of this methods?
* cWorld::RemovePlayer
* cClientHandle::SendDisconnect

Reply:
Looking in `cClientHandle`, it seems to already do `SetIsTicking(false)`:
https://github.com/cuberite/cuberite/blob/5a2163d7e6ffa0c5c568be41257fb726d452b3b9/src/ClientHandle.cpp#L185
That  would suggest the player entity is being destroyed before `cClientHandle::Destroy` is called.

Reply:
Tested both methods, didn't work :(

Reply:
Okay closing this and will open an issue, can reproduce it now easily.